bin/rename.js

```diff
@@ -39,25 +39,32 @@ const findReplace = (findString, replaceString) => {
 
   try {
     const changes = replace.sync(options);
+
+    consoleOutput(fgGreen, '');
+    consoleOutput(fgGreen, '------------');
     consoleOutput(fgGreen, `${findString}-> ${replaceString}. Modified files: ${changes.join(', ')}`);
   } catch (error) {
+
+    consoleOutput(fgMagenta, '');
+    consoleOutput(fgMagenta, '------------');
     console.error('Error occurred:', error);
   }
 };
 
 
 // Main script
-consoleOutput(fgGreen, 'The script will uniquely set up your theme.');
+consoleOutput(fgGreen, 'Welcome to Boilerplate readme script. The script will uniquely set up your theme.');
```

Comment:
> Welcome to Boilerplate readme script

You meant `Welcome to Boilerplate rename script` ? 😄

Reply:
maybe :D